update/sync/coordinator.go

```diff
@@ -60,12 +78,12 @@ func (ss *syncState) SyncAllState(epoch uint32) error {
 	mutErr := sync.Mutex{}
 
 	go func() {
-		err := ss.tries.SyncTriesFrom(meta, time.Hour)
-		if err != nil {
-			mutErr.Lock()
-			errFound = err
-			mutErr.Unlock()
-		}
+		//err := ss.tries.SyncTriesFrom(meta, time.Hour)
```

Comment:
no need to comment this. there is a nil implementation.

Reply:
you commented the code here